src/commands/Games/higherLower.ts

```diff
@@ -0,0 +1,246 @@
+
+			switch (emojiSelected) {
+				case ReactionEmoji.HIGHER:
+					game.running = await (oldNum < game.number ? this.win(game, message) : this.loss(game, message));
+					break;
+
+				case ReactionEmoji.LOWER:
+					game.running = await (oldNum > game.number ? this.win(game, message) : this.loss(game, message));
+					break;
+
+				case ReactionEmoji.CASHOUT:
+				case ReactionEmoji.CANCEL:
+					await this.end(game, message, emojiSelected === ReactionEmoji.CASHOUT);
+					game.running = false;
+					break;
+			}
+
+			if (game.running) game.turn++;
+		}
+	}
+
+	private async win(game: HigherLowerGameData, message: KlasaMessage) {
+		console.log('in win arg');
+		const { author: user } = message;
+		await game.gameMessage.reactions.removeAll();
+		// TODO(Quantum): Winning event
+		await game.gameMessage.edit({
+			content: '',
+			embed: new MessageEmbed()
+				.setColor(getColor(message as KlasaMessage))
+				.setTitle(`Your won!`)
+				.setDescription(`You did it! The number was ${game.number}. Want to continue? ${this.calculateWinnings(game.wager, game.turn)}${Emojis.Shiny} are on the line`)
+				.setFooter('Act fast! You don\'t have much time')
+		});
+		console.log('filtering emotes');
+		const reactArray = Object.values(ReactionEmoji).filter(e => e !== ReactionEmoji.HIGHER && e !== ReactionEmoji.LOWER && e !== ReactionEmoji.CASHOUT);
+		for (const emoji of reactArray) {
+			await game.gameMessage.react(emoji);
+		}
+		console.log('GETTING EMOTES');
+		const reactionMap = await game.gameMessage.awaitReactions((r, u) => reactArray.includes(r.emoji.name) && u.id === user.id, { max: 1, time: 10000 });
+		await game.gameMessage.reactions.removeAll();
+
+		const whatToDo = reactionMap.size === 0
+			? endingAction.TIMEOUT
+			: reactionMap.first()!.emoji.name === ReactionEmoji.OK
+				? endingAction.PLAY
+				: endingAction.STOP;
+		switch (whatToDo) {
+			case endingAction.TIMEOUT:
+				await game.gameMessage.edit('Prompt timed out. Cashing out winnings...', { embed: null });
+			case endingAction.STOP:
+				await this.end(game, message, true);
+				break;
+			case endingAction.PLAY:
+				await game.gameMessage.edit('Alright. Starting new round', { embed: null });
+				break;
+		}
+		return whatToDo === endingAction.PLAY;
+	}
+
+	private async loss(game: HigherLowerGameData, message: KlasaMessage) {
+		// TODO(Quantum): Losing event
+		const { number, wager, turn, gameMessage } = game;
+		await gameMessage.reactions.removeAll();
+		let losings = wager;
+		if ((Math.random()) < 0.001) {
+			losings += this.calculateWinnings(wager, turn - 1);
+			await message.author.settings.decrease(UserSettings.Money, losings);
+		}
+		await gameMessage.edit(new MessageEmbed()
+			.setColor(getColor(message))
+			.setTitle(`You lost :\(`)
+			.setDescription(`You didn't quite get it. The number was ${number}. You lost ${losings}${Emojis.Shiny}`)
+			.setFooter('Better luck next time!'));
+		console.log('mesage should be edited');
```

Comment:
```suggestion
		console.log('message should be edited');
```

At least spell your console logs correctly smh